src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx

```diff
@@ -73,6 +74,7 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
       <PhoneNumberInput
         mt={4}
         phoneNumber={values.phone}
+        optional
```

Comment:
does this mean that it will be always optional? 👀

Reply:
Oooooh, it shouldn't! I've fixed the logic to pass `optionalPhoneNumber`.